stan/math/opencl/prim/col.hpp

```diff
@@ -12,20 +12,19 @@ namespace math {
  * Return the specified column of the specified kernel generator
  * expression using start-at-1 indexing.
  *
- * This is equivalent to calling <code>m.col(i - 1)</code> and
- * assigning the resulting expression to a matrix_cl representing
- * a column vector.
+ * This is equivalent to calling <code>x.col(i - 1)</code> and
+ * assigning the resulting expression to a column vector.
```

Comment:
That is a bit confusing. Kernel generator expressions are not assignable to (Eigen) column vectors. I prefere previous version of this comment that mentions destinationn type is matrix_cl

Reply:
This was copied from the col.hpp on eigen matrices. Since matrix_cl does not have this .col() function, I will just remove this sentence.